scss/navbar.scss

```diff
@@ -19,3 +19,9 @@
   content: url('./assets/wolox_logo.svg');
   min-width: 200px;
 }
+
+@media screen and (max-width: 1024px) {
+  .header-title {
+    visibility: hidden;
```

Comment:
what you want here is `display: none;` because display none makes the element disappear and stop taking up space